examples/pipelines/controlnet-interior-design/pipeline.py

```diff
@@ -82,4 +87,18 @@
 pipeline.add_op(segment_images_op, dependencies=caption_images_op)
 pipeline.add_op(write_to_hub_controlnet, dependencies=segment_images_op)
 
-client.compile_and_run(pipeline=pipeline)
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--local", action="store_true")
+    if parser.parse_args().local:
+        compiler = DockerCompiler()
+        extra_volumes = [
+            "$HOME/.config/gcloud/application_default_credentials.json:/root/.config/gcloud/"
+            "application_default_credentials.json:ro"
+        ]
+        compiler.compile(pipeline=pipeline, extra_volumes=extra_volumes)
+        logger.info("Run `docker-compose up` to run the pipeline.")
+
+    else:
+        client = Client(host=PipelineConfigs.HOST)
+        client.compile_and_run(pipeline=pipeline)
```

Comment:
Let's not include this. This should be handled by the `fondant` command in the end.